lib/protoflow-blocks/src/blocks/io.rs

```diff
@@ -133,6 +141,9 @@ pub mod io {
     mod encode_hex;
     pub use encode_hex::*;
 
+    mod decode_hex;
+    pub use decode_hex::*;
+
```

Comment:
These module declarations should be in alphabetical order.

Reply:
done